CRM/Event/Form/SelfSvcTransfer.php

```diff
@@ -399,14 +380,15 @@ public function participantTransfer($participant) {
       'entity_table' => 'civicrm_event',
     ];
     $eventDetails['location'] = CRM_Core_BAO_Location::getValues($locParams, TRUE);
-    $toEmail = $contactDetails[$participant->contact_id]['email'] ?? NULL;
+    $toEmail = $contactDetails['email'] ?? NULL;
     if ($toEmail) {
       //take a receipt from as event else domain.
-      $receiptFrom = $domainValues['name'] . ' <' . $domainValues['email'] . '>';
+      $receiptFrom = CRM_Core_BAO_Domain::getNameAndEmail(FALSE, TRUE);
+      $receiptFrom = reset($receiptFrom);
```

Comment:
OK, so this removes the last remaining reference to `$domainValues` (in this function) - making all the 15 SLOC silliness above unnecessary. :+1: